src/Configuration.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\BetterReflection;
+
+use PhpParser\Lexer\Emulative;
+use PhpParser\Parser;
+use PhpParser\ParserFactory;
+use Roave\BetterReflection\NodeCompiler\CompileNodeToValue;
+use Roave\BetterReflection\Reflector\ClassReflector;
+use Roave\BetterReflection\Reflector\FunctionReflector;
+use Roave\BetterReflection\SourceLocator\Ast\Locator as AstLocator;
+use Roave\BetterReflection\SourceLocator\Ast\Strategy\AstConversionStrategy;
+use Roave\BetterReflection\SourceLocator\Ast\Strategy\NodeToReflection;
+use Roave\BetterReflection\SourceLocator\Type\AggregateSourceLocator;
+use Roave\BetterReflection\SourceLocator\Type\AutoloadSourceLocator;
+use Roave\BetterReflection\SourceLocator\Type\EvaledCodeSourceLocator;
+use Roave\BetterReflection\SourceLocator\Type\PhpInternalSourceLocator;
+use Roave\BetterReflection\SourceLocator\Type\SourceLocator;
+use Roave\BetterReflection\Util\FindReflectionOnLine;
+
+// @TODO should the naming be swapped around? `new BetterReflection()` sounds MUCH cooler!
+final class Configuration implements BetterReflection
```

Comment:
After finishing it, I saw that the interface is not really needed. In my opinion, this is OK as-is, so the `implements` keyword should disappear.